doc/user/content/third-party/dbt.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Using dbt"
+description: "Get details about using Materialize with dbt"
+menu:
+  main:
+    parent: 'third-party'
+---
+
+You can use [dbt] to easily create views and materialized views in your `materialized`
+instance. To do so, all you need to do is install our [dbt-materialize] adapter.
+
+*Note*: The [dbt-materialize] adapter is still a work in progress and is not yet suitable
+for production use-cases. You can check its limitations and track our progress
+[here](https://github.com/MaterializeInc/materialize/issues/5462).
+
+## Limitations
```

Comment:
I really like the heading "What's missing?" from the overview doc––it made this section's presence make much more sense. This section is largely redundant with the note above, though––to get the best of all the pieces, I suggest we:
- Get rid of the note above
- Change this heading to "What's missing?"
- Add a warning that simply says:
    > Materialize does not offer production-level support for [tool]. 
- Add a lead in sentence that says something like:
    > Materialize will not offer production-level support for [tool] until...

Reply:
This format is great--thank you!